lib/cldr.js

```diff
@@ -149,6 +159,22 @@ Cldr.prototype = {
         }
         return parentLocaleId;
     },
+    
+    resolveWindowsZonesByTimezone: function (timezone) {
```

Comment:
Could you rename this to `extractWindowsZonesByTimezone` and move it down a bit so it's next to the other user-facing `extract`-prefixed methods?